buildstock_query/db_schema/comstock_oedi.toml

```diff
@@ -0,0 +1,32 @@
+[table_suffix]
+baseline = "_metadata"
+timeseries = "_by_state"
+upgrades = "_upgrades"
+
+[column_prefix]
+characteristics = "in."
+output = "out."
+
+[column_names]
+building_id = "bldg_id"
+sample_weight = "weight"
+sqft = "in.sqft"
+timestamp = "timestamp"
+completed_status = "applicability"
+unmet_hours_cooling_hr = ""
+unmet_hours_heating_hr = ""
+fuel_totals = ["out.district_cooling.total.energy_consumption",
+               "out.district_heating.total.energy_consumption",
+               "out.electricity.total.energy_consumption",
+               "out.natural_gas.total.energy_consumption",
+               "out.other_fuel.total.energy_consumption",
+               "out.site_energy.total.energy_consumption"
+               ]
+
+[completion_values]
+success = "true"
+fail = ""
+unapplicable = "false"
+
+[structure]
+unapplicables_have_ts = "false"
```

Comment:
The telescope tool that uploads the result to OEDI automatically copies baseline ts to unapplicable buildings, so I think this should be true. Unless I am mistaken about how this works for ComStock.

Reply:
changed to true